tests/operators/test_python.py

```diff
@@ -830,19 +830,6 @@ def f():
         with pytest.raises(CalledProcessError):
             self._run_as_operator(f)
 
-    def test_python_2(self):
-        def f():
-            {}.iteritems()  # pylint: disable=no-member
-
-        self._run_as_operator(f, python_version=2, requirements=['dill'])
-
-    def test_python_2_7(self):
```

Comment:
I do not think we should remove those. We've never decided on dropping 2.7 as a way to exxecute the Python operator. That was always an "escape hatch" for people who wanted to still execute their python 2.7 code even if it is not used/needed by Airflow itself. If we want to drop it, it should be discussed at the devlist because it has serious implications for our users who **might** still have some python 2.7 code around.

Besides having to install the 2_7 library in the image, It has completely no impact on the code of Airflow, so there is no particular reason to remove it IMHO. It will not impact the way we are developing Airflow, it does not change any tooling support, it is just a way to nicely execute external python 2.7 code from within Airflow codebase.

Reply:
Thanks for the feedback! Totally makes sense.